src/components/core/DpUpload/DpUploadFiles.stories.mdx

```diff
@@ -0,0 +1,29 @@
+import { Meta } from '@storybook/addon-docs'
+import DpUploadFiles from './DpUploadFiles'
+
+<Meta
+    title="Components/UploadFiles"
+    component={DpUploadFiles}
+/>
+
+# UploadFiles
+
+The UploadFiles component can be upload a single file or multiple files.
+
+The `allowedFileTypes` Prop allows us to choose the type of files that we want to upload.
```

Comment:
I see that props within DpUploadFiles.vue are not documented equally helpful. Apart from the need to document all props, i'd opt for examples with...

- only allow zip uploads
- max file size is limited
- @file-remove and @upload-success is used

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/518/commits/2999afd2b3761940bacc0dabf0dee335914801bc
Is this okay?